content/docs/command-reference/config.md

```diff
@@ -352,6 +352,15 @@ experiments or projects use a similar structure.
   files will be stored, by default in `.dvc/tmp/index`. This may be necessary
   when using DVC on NFS or other mounted volumes.
 
+### hydra
+
+- `hydra.enabled` - enables
+  [Hydra Integration](/docs/user-guide/hydra-integration).
```

Comment:
What about calling it `Hydra Config Integration`? Or something with `Compose` in it? I worry `Hydra Integration` is too generic and is confusing when combined with the other Hydra-supported features.

Reply:
`Hydra Composition` makes sense?